src/Sinks/SerilogVisitor.cs

```diff
@@ -0,0 +1,73 @@
+namespace Serilog.Sinks.Fluentd.Core.Sinks
+{
+    using System;
+    using System.Collections.Generic;
+    using Serilog.Events;
+
+    public class SerilogVisitor
+    {
+        public void Visit(IDictionary<string, object> state, string name, LogEventPropertyValue value)
+        {
+            var returnValue = this.Visit(value);
+            state[name] = returnValue;
```

Comment:
Whoever came up with this is bloody brilliant ;)

Reply:
THanks